.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            "tab"
```

Comment:
Did you choose to use tabs or was this a preset?

Reply:
This was a preset